api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnReportEndDateDateCalculation.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.module.eptsreports.reporting.calculation.generic;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+import org.joda.time.Interval;
+import org.joda.time.Years;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResult;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.module.eptsreports.reporting.calculation.AbstractPatientCalculation;
+import org.openmrs.module.eptsreports.reporting.calculation.BooleanResult;
+import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
+import org.openmrs.module.reporting.common.Birthdate;
+import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AgeOnReportEndDateDateCalculation extends AbstractPatientCalculation {
```

Comment:
@cfaife can you please put some javadocs on the propose of this calculation.... will be helpful tomorrow

Reply:
Ok sure I will

Reply:
done